pkg/metrics/statsd/azure/metrics.go

```diff
@@ -0,0 +1,66 @@
+package azure
```

Comment:
The azure and k8s implementations imply an unnecessary additional allocation and GC on every API call - this can be removed by making each tracer a field of metrics.Statsd.  Consider whether to use an embedded ("non-named") field or not.

Reply:
I am completely wrong, hehe.  There are 2 tracer objects statically allocated per statsd for k8s, and 1 for azure; none per API call.  Leave it all as it is.